apps/chat/models.py

```diff
@@ -127,3 +127,17 @@ def get_attached_files(self):
         if file_ids:
             return File.objects.filter(team=self.chat.team, external_id__in=file_ids)
         return []
+
+
+class ThreadToolResources(BaseModel):
```

Comment:
Sorry, I should have made this suggestion earlier. What about renaming this to `ChatAttachments`?

Also just to mention that the next ticket is to support downloading attachments generated by the AI and I think we can use the same model. Just want to make sure we get it right now so we don't have to immediately change it.